tests/test_folium.py

```diff
@@ -473,6 +456,24 @@ def test_topo_json(self):
         assert bounds == [[41.99187135900012, -124.56617536999985],
                           [46.28768217800006, -116.46422312599977]], bounds
 
+    def test_topo_json_smooth_factor(self):
+        """Test topojson smooth factor method."""
+        self.map = folium.Map([43, -100], zoom_start=4)
+
+        # Adding TopoJSON as additional layer.
+        path = os.path.join(rootpath, 'or_counties_topo.json')
+        self.map.choropleth(geo_path=path,
+                            topojson='objects.or_counties_geo',
+                            smooth_factor=0.5)
+
+        out = self.map._parent.render()
+
+        # Verify TopoJson
+        topo_json = [val for key, val in self.map._children.items()
+                     if isinstance(val, TopoJson)][0]
+        topojson_str = topo_json._template.module.script(topo_json)
+        assert ''.join(topojson_str.split())[:-1] in ''.join(out.split())
```

Comment:
This is fine. Thanks!

Can you also add a notebook example showing the new feature? (That is optional we don't need that to merge.)